packages/scripts/build-package.ts

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env tsx
 
-import { rm, cp } from "node:fs/promises";
+import { rm, cp, access } from "node:fs/promises";
 import { join } from "node:path";
 import { totalist } from "totalist";
 import { build, context } from "esbuild";
 
 const args = process.argv.slice(2);
 const watch = args.includes("--watch");
+const noGeneratedAsEntries = args.includes("--no-generated-as-entries");
```

Comment:
This is a little bit quirky... but we have 2 use cases here:
- `esbuild` should ignore the generated directory in `@webstudio-is/prisma-client`, it contains more than just `.ts` files and `esbuild` trips over the shape of `.d.ts` within it. In this case, we only want to copy-over the generated directory into the dist dirs
- in other packages though... the generated files are used as entrypoints~ so they should be handled by `esbuild` and not ignored

Reply:
Can't we just ignore .d.ts here? Shouldn't break anything

Reply:
That was something I wanted to do at first but then I realized that there are still other files there that we need to copy over to the dist directory. So it's just easier to copy over the whole thing and ignore it for esbuild since it literally doesn't have to touch it at all.